examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OverviewFragment.kt

```diff
@@ -92,7 +92,9 @@ class OverviewFragment : Fragment(), OfferingCardAdapter.OfferingCardAdapterList
 
         val offeringCardTransitionName = getString(R.string.offering_fragment_transition)
         val extras = FragmentNavigatorExtras(cardView to offeringCardTransitionName)
-        val directions = OverviewFragmentDirections.actionOverviewFragmentToOfferingFragment(offering)
+//        val directions = OverviewFragmentDirections.actionOverviewFragmentToOfferingFragment(offering)
+        // uncomment line below and comment line above to test deprecated purchase methods
+        val directions = OverviewFragmentDirections.actionOverviewFragmentToDeprecatedOfferingFragment(offering)
```

Comment:
Should the bottom line be commented out? 😇

Reply:
yep definitely will do before properly opening! thinking about making this a runtime thing and adding a checkbox. and we can prob remove these before releasing v6..